pkg/ec2helper/ec2helper.go

```diff
@@ -909,11 +912,39 @@ func (h *EC2Helper) ParseConfig(simpleConfig *config.SimpleInfo) (*config.Detail
 		}
 	}
 
+	// Add simple-ec2 tags to created resources
+	simpleEc2Tags := getSimpleEc2Tags()
```

Comment:
may be all the logic related to tags should be in one place for readability like `https://github.com/awslabs/aws-simple-ec2-cli/tree/main/pkg/tag`? 

Irrespective of that change, I think the following changes will bring clarity:
-  [method](https://github.com/awslabs/aws-simple-ec2-cli/blob/main/pkg/tag/tag.go#L22) `GetTags` should be renamed to `GetSimpleEc2Tags` for clarity. 
- (nit) rename variables to differentiate usage of `simpleEc2Tags` here v/s [here](https://github.com/awslabs/aws-simple-ec2-cli/blob/fe6fcc810591133d6af72dd427fda0b1a99a6b87/pkg/ec2helper/ec2helper.go#L792)

Reply:
good call-- definitely a lot of refactor opportunities here. I'll address this in next revision

Reply:
- renamed methods and vars for clarity
- using the StringToString map cleared up a lot of this logic